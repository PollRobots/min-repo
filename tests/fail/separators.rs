use llvm_cov_358_min_repo::fizzbuz;

pub fn main() {
    let _ = fizzbuz! {
        1 . 2| fizz; 4: buzz  fizz
    };
}
