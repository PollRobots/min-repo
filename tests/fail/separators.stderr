error: Expecting ','
 --> tests/fail/separators.rs:5:11
  |
5 |         1 . 2| fizz; 4: buzz  fizz
  |           ^

error: Expecting ','
 --> tests/fail/separators.rs:5:14
  |
5 |         1 . 2| fizz; 4: buzz  fizz
  |              ^

error: Expecting ','
 --> tests/fail/separators.rs:5:20
  |
5 |         1 . 2| fizz; 4: buzz  fizz
  |                    ^

error: Expecting ','
 --> tests/fail/separators.rs:5:23
  |
5 |         1 . 2| fizz; 4: buzz  fizz
  |                       ^

error: Expecting ','
 --> tests/fail/separators.rs:5:31
  |
5 |         1 . 2| fizz; 4: buzz  fizz
  |                               ^^^^
